CONTRIBUTING.md

```diff
@@ -102,7 +102,7 @@ After making changes to the base files (and updating imports when adding new fil
 <details>
 <summary>Create a pull request to Backpack Foundations</summary>
 
-For anything non-trivial, we strongly recommend speaking to somebody from Koala before starting work on a PR. This lets us pass on any advice or knowledge we already have about the work you're proposing. It might even be something we're already working on. After this, follow the steps below.
+For anything non-trivial, we strongly recommend speaking to somebody from Clover before starting work on a PR. This lets us pass on any advice or knowledge we already have about the work you're proposing. It might even be something we're already working on. After this, follow the steps below.
```

Comment:
We should possibly add Donburi here too or just change to be

```suggestion
For anything non-trivial, we strongly recommend sspeaking to the Backpack owners before starting work on a PR. This lets us pass on any advice or knowledge we already have about the work you're proposing. It might even be something we're already working on. After this, follow the steps below.
```